common/app/model/meta.scala

```diff
@@ -723,3 +724,17 @@ object Tags {
     Tags(apiContent.tags.toList map { Tag.make(_) })
   }
 }
+
+object AdUnitMaker extends Logging {
+
+  val knownBadAdUnits = Seq("article")
+
+  def make(pageId: String, adUnitSuffix: String): String = {
+    if (knownBadAdUnits.contains(adUnitSuffix)) {
```

Comment:
Also I noticed that [many ad units end with `article`](https://s3-eu-west-1.amazonaws.com/aws-frontend-store/PROD/commercial/dfp/active-ad-units.csv?X-Amz-Date=20160914T132408Z&X-Amz-Expires=300&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Signature=f6b0d69a76baf2c547f7aa96db236a0fb8dd8c19aa1f761f99b1fce174313841&X-Amz-Credential=ASIAI33KANIUE7UE33UA/20160914/eu-west-1/s3/aws4_request&X-Amz-SignedHeaders=Host&x-amz-security-token=FQoDYXdzEKf//////////wEaDJAn37wmY01gZS01ZyLaAo4MR%2BpRhKfeu7Ikip1GfY4bge88Pzh74zqOz7pDoMY0P759MCoG9jkcauZGbs90kLrW28MCXEtU%2BDhpi0Q9oe6U%2Bh4DI1gkS4Nz0HL3ZoFsKIozkLtPE6GFWiHh%2BOd5H%2BEMRq16JWDFci1f82CjdmM2FwoivnPsQMgcISa0Z1dObH//z481tRxlmOCfKWDqW9Jf48jxtsvg4kKnA7zMlsZuxInQ0DurxP9zhmdBzjdhe06f/Z9LGEqH2In2%2BEuR0u0IUDytBHl9G1H8wsW68j0fAccsqNhLLIq54ZLC2qQYDQbUqi7mnVL76BkfGNv4wj0q1Khgs8B9qvx%2BUVA59i/TZotkXGofd/%2BzYJnCkhByoRdDu0hrkAT5%2BZ7xJTvaTb/zsxBCyhU1vaEbfvaJucK6PWG2RT/tohl5Qej1LhEPP4bq2L%2BQEnaqn%2B%2BapIeDYnhb4v1F4uwoTOoo5Z7lvgU%3D), will they generate false positives?